testnet_deployed.json

```diff
@@ -1,6 +1,7 @@
 {
 	"chainId": 5,
 	"networkId": 10,
+  "minimumBucketDepth": 16,
```

Comment:
Why is this in this file?

Reply:
I discussed this with Abel, it seems like this would be different for different cases and is passed in postage stamp contract which is why it is in file